manifests/vhost.pp

```diff
@@ -2111,7 +2111,7 @@
     false => $name,
   }
 
-  if ! $use_servername_for_filenames and $name != $normalized_servername {
+  if ! $use_servername_for_filenames and $servername != $normalized_servername {
```

Comment:
This looks correct to me. I guess in my testing I didnt' specify a servername so they happened to match.

Reply:
Btw, I'm reading this again and what I based it on was this part:
https://github.com/puppetlabs/puppetlabs-apache/blob/337e8b00b2a54f503e2ab0fa5c16d7a3a648c302/manifests/vhost.pp#L2106-L2112

So right now it uses `$name` for `$filename`, but after it will use `$normalized_servername` which is a change. Maybe I should have done it otherwise and determined `$v8_filename` instead and compared that.

Reply:
Sorry, as I see now the check between $name/$normalized_servername and between $servername/$normalized_servername are both bogus in that case.

While I see no problem at all to use a normalized-servername as the filename for the apache-configsnipped, I understand the reasoning in rolling that change in stages and issue warnings.

However, I really would like to have a possibility to "defuse" these warnings on a global level. While still messy, maybe we could lookup the $use_servername_for_filenames and $use_port_for_filenames from hiera? In that case it would be at least possible to enable both parameters on one place.